EIPS/eip-7742.md

```diff
@@ -33,67 +33,99 @@ This EIP also changes how the EL sources the current blob target value for two r
 The data facility introduced via EIP-4844 adds blobs to Ethereum blocks, which are simply fixed sets of data that can be
 included in the canonical chain but have no execution semantics (cf. `calldata` in an Ethereum transaction).
 
-The protocol specifies a maximum allowed blob count per block to prevent DoS vectors via the abuse of this data facility.
+The protocol specifies a maximum allowed blobs per block to prevent DoS vectors via the abuse of this data facility.
 The protocol also maintains an [EIP-1559](./eip-1559.md)-like "target" value for an intended running average amount of blob throughput per
 unit time. Blob usage is compared against this target to influence a "blob base fee" to administer allocation of this
 resource to users of the Ethereum protocol.
 
 Both of these values are currently hard-coded in the EL after EIP-4844 and the blob maximum is separately hard-coded in
 the CL following EIP-4844. This EIP proposes a set of changes to uncouple these values across the CL and EL to make development
-and deployment of changes to the blob count easier.
+and deployment of changes to the blobs per block easier.
 
 #### Maximum blobs per block
 
-The blob maximum is verified in the CL node and the EL inherits this verification during the consistency check of the
-versioned hashes corresponding to each blob as specified by the Engine API. Because of this, the strict check specified
-by EIP-4844 is unnecessary.
+The blob maximum is verified in the CL node and the EL inherits this verification during the consistency check of the versioned hashes corresponding to each blob as specified by the Engine API. Because of this, the strict check specified by EIP-4844 is unnecessary.
 
-#### Target amount of blobs per block
+For optimistically synced blocks from peer nodes, EL may further assume that the entire ancestor EL peer synced chain blocks correctly obeys this condition by the canonicality and correctness of the block they backfilling from as instructed by the CL.
 
-The target is currently specified as a fixed value in relation to the blob count. The Ethereum community intends to increase
-the blob parameters as part of its scaling strategy and the ability to have a more flexible target value in relation to
-the blob max is desirable to reduce rigidity in this protocol parameter.
+Hence we entirely deprecated the `MAX_BLOB_GAS_PER_BLOCK` checks from [EIP-4844](./eip-4844.md) in the EL while letting the maximum blobs per block constraints in the CL stay at `6` equivalent to `MAX_BLOB_GAS_PER_BLOCK`.
```

Comment:
I would say something like this as it is confusing when we later suggest raising the max below

```suggestion
Hence we entirely deprecated the `MAX_BLOB_GAS_PER_BLOCK` checks from [EIP-4844](./eip-4844.md) in the EL while letting the maximum blobs per block constraint to be checked by the CL.
```